### ISSUE TITLE
Add missing alt text to ENIGMA logo

### ISSUE THREAD

**Issue by pushpaktiwarii** [pushpaktiwarii/enigma#1]
The ENIGMA main logo does not have an alt attribute. Add alt="Enigma Logo" for accessibility and SEO.
